.github/workflows/ci.yml

```diff
@@ -8,6 +8,20 @@ on:
     branches: [main]
     types: [opened, reopened, synchronize]
 jobs:
+  check_if_build:
```

Comment:
Add a name to the job in Title Case

https://github.com/tailcallhq/tailcall/blob/main/.github/contributing.md#making-changes

Reply:
done